src/apps/frontend/services/auth.service.ts

```diff
@@ -19,11 +20,13 @@ export default class AuthService extends APIService {
   login = async (
     username: string,
     password: string,
-  ): Promise<ApiResponse<AccessToken>> =>
-    this.apiClient.post('/access-tokens', {
+  ): Promise<ApiResponse<AccessToken>> =>{
+    const response = await this.apiClient.post('/access-tokens', {
       username,
       password,
     });
+    return new ApiResponse(new AccessToken(response.data as JsonObject))
```

Comment:
We need to avoid usage of `as`, it possibly surfaces that somewhere, the typing is not done right.

Can we look into `this.apiClient.post` to see if it can return `JsonObject`?